action.yml

```diff
@@ -132,13 +132,35 @@ runs:
       shell: bash
       working-directory: ${{ inputs.working-directory }}
       run: |
+        function print_assignment {
+          echo "$(echo "$1=$2" | tr '\n' ' ')"
+        }
```

Comment:
I think you can simplify this by accepting the value on `stdin` instead of `$2`:

```sh
print_assignment() {
  printf '%s=' "$1"
  tr '\n' ' '
}
```

And use it like,

```sh
print_assignment "stack-query-arguments" <<EOM
${{ inputs.stack-query-arguments }}
EOM
```